package.json

```diff
@@ -23,7 +23,7 @@
     "css-loader": "^1.0.0",
     "enzyme": "^3.6.0",
     "enzyme-adapter-react-16": "^1.5.0",
-    "eslint": "^5.4.0",
+    "eslint": "^5.16.0",
```

Comment:
Why do we need this version bump?

Reply:
I installed an eslint extension in Brackets, and I think it bumped the version number up. Will remove this